9c7718cddb229150791d8f8d13961979/update/sys_script_b91e6805dba29150791d8f8d13961933.xml

```diff
@@ -29,7 +29,7 @@
         <role_conditions/>
         <script><![CDATA[(function executeRule(current, previous /*null when async*/) {
 
-	var matches = current.text.match(/(?:<@)(.*?)(?=>\s?\+\+)/g);
+	var matches = current.text.match(/(?:<@)(.*?)(?=>\s*\+\+)/g);
```

Comment:
this doesn't take any spaces into account between the two plusses

Reply:
changing the regex to `/(?:<@)(.*?)(?=>\s*\+){2}/` should be pretty inclusive.

e.g., https://regex101.com/r/O3XsrE/2

Reply:
regex in this commit will fail on the top two use cases, as seen here https://regex101.com/r/nLj5fl/1